factory/installScripts/edge/default.sh

```diff
@@ -3,7 +3,6 @@
 # Microsoft offers a debian package, here we're adding the package list and then installing the specific version we want to install.
 apt-get update \
   && apt-get install --no-install-recommends -y \
```

Comment:
I think you can remove this too. we aren't installing anything now.

Reply:
missed that 😬  . OK fixed in 5b74cb6